Blockzilla/SiriShortcuts.swift

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Intents
+import IntentsUI
+
+class SiriShortcuts {
+    enum activityType: String {
+        case erase
+        case eraseAndOpen = "org.mozilla.ios.Klar.eraseAndOpen"
+        case openURL
+    }
+    
+    func getActivity(for type: activityType) -> NSUserActivity? {
+        switch type {
+        case .erase:
+            print("erase")
```

Comment:
Is this part still a work in progress? We shouldn't be shipping debug statements.